config/routes.rb

```diff
@@ -26,6 +26,7 @@
 
   get "search", to: "search#index"
 
+  get "my/favorite-gemfiles", to: "my#favorite_gemfiles"
```

Comment:
Would prefer to keep RESTful routing with the standard actions.  Although love the namespacing.  
```suggestion
  namespace :my do 
    resources :favorites, only: [:index]
  end
```